TACC/tup-ui
# Fixed create account and get help link on login

## Overview:
Set Create Account  links to https://accounts.tacc.utexas.edu/register for Create Account and Account Help to be a new ticket at https://dev.tup.tacc.utexas.edu/about/help/

## Related:

- [TUP-407](https://jira.tacc.utexas.edu/browse/TUP-407)

## Changes:

- Changed link for Create Account to open in new tab at https://accounts.tacc.utexas.edu/register 
- Changed link for Account Help to open in a new tab at https://dev.tup.tacc.utexas.edu/about/help/
## Testing:

1. Make sure you're logged out of your TACC account
2. Click Log In
3. Click Create Account and make sure it goes to  https://accounts.tacc.utexas.edu/register 
4. Click Account Help and make sure it goes to  https://dev.tup.tacc.utexas.edu/about/help/


## UI:
Create Account
<img width="1568" alt="Screen Shot 2023-02-13 at 9 28 28 AM" src="https://user-images.githubusercontent.com/96220951/218500066-6a1262a5-f9e1-4a1a-ae9d-4c704e865ee7.png">

Account Help
<img width="1568" alt="Screen Shot 2023-02-13 at 9 28 56 AM" src="https://user-images.githubusercontent.com/96220951/218500192-b612b8a0-3149-480a-b490-449c1e3ec85f.png">
